### PR TITLE
[4.0] Update browser versions for autoprefixer

### PR DESCRIPTION
**Pull request by C-Lodder**
Pull Request for Issue # .

### Summary of Changes

- Update autoprefixer to v7.1.3
- Update browserlist to 2 latest versions
- Recompile

### CODE REVIEW

File: grunt-settings.yaml [hunk 1]
```diff
@@ -1,11 +1,12 @@
-#### Browser Version is used for automatically prefixing the stylesheets. IMPORTANT: Joomla 4 supports only the latest 2 versions of each browser!!!
+#### Browser Version is used for automatically prefixing the stylesheets.
+#### IMPORTANT: Joomla 4 only supports the latest 2 versions of each browser, with exceptions for Safari and IE
```
Comment (brianteeman) [inline grunt-settings.yaml:2]: isnt it current and previous two versions?

File: templates/aurora/css/template.css [hunk 2]
```diff
@@ -8489,7 +8489,7 @@ joomla-alert {
...
-    grid-template-areas: ". head head head head ." ". banner banner banner banner ." ". top-a top-a top-a top-a ." ". top-b top-b top-b top-b ." ". main main main main ." ". bot-a bot-a bot-a bot-a ." ". bot-b bot-b bot-b bot-b ." ". footer footer footer footer ." ". debug debug debug debug .";
+    grid-template-areas: ". head head head head ."
 ". banner banner banner banner ."
 ". top-a top-a top-a top-a ."
 ". top-b top-b top-b top-b ."
 ". main main main main ."
 ". bot-a bot-a bot-a bot-a ."
 ". bot-b bot-b bot-b bot-b ."
 ". footer footer footer footer ."
 ". debug debug debug debug .";
```
Comment (brianteeman) [inline templates/aurora/css/template.css:8492]: looks like some odd characters here

File: templates/aurora/css/template.css [hunk 3]
```diff
@@ -8508,7 +8508,7 @@ joomla-alert {
...
-        grid-template-areas: ". head head head head ." ". banner banner banner banner ." ". main main main main ." ". top-a top-a top-a top-a ." ". top-b top-b top-b top-b ." ". bot-a bot-a bot-a bot-a ." ". bot-b bot-b bot-b bot-b ." ". footer footer footer footer ." ". debug debug debug debug ."; } }
+        grid-template-areas: ". head head head head ."
 ". banner banner banner banner ."
 ". main main main main ."
 ". top-a top-a top-a top-a ."
 ". top-b top-b top-b top-b ."
 ". bot-a bot-a bot-a bot-a ."
 ". bot-b bot-b bot-b bot-b ."
 ". footer footer footer footer ."
 ". debug debug debug debug ."; } }
```
Comment (brianteeman) [inline templates/aurora/css/template.css:8511]: add the same odd characters here

Comment (C-Lodder) [reply templates/aurora/css/template.css:8511]: Not an issue tbh. Just need to convert my line ending to Unix on Git

**Comment by C-Lodder** [comment]
Will be updating the grunt-settings to support only the **latest** version of all major browsers, not the last 2 or 3.

**Comment by dgrammatiko** [comment]
@C-Lodder is it safe to rely only on the latest version?

**Comment by C-Lodder** [comment]
@dgt41 - The latest stable, yes. BS do it, so considering we're using some of their components, we need to aswell.

**Pull request closed without merging**